iphone/Maps/Classes/LocaleTranslator.mm

```diff
@@ -0,0 +1,24 @@
+#include "LocaleTranslator.h"
+
+namespace locale_translator
+{
+
+string bcp47ToTwineLanguage(NSString const * bcp47LangName)
+{
+  if (bcp47LangName == nil || [bcp47LangName length] < 2)
+    return string("");
+
+  if ([bcp47LangName isEqualToString:@"zh-CN"] || [bcp47LangName isEqualToString:@"zh-CHS"]
+      || [bcp47LangName isEqualToString:@"zh-SG"])
+  {
+    return string("zh-Hans"); // Chinese simplified
```

Comment:
What do you mean?
```return "zh-Hans";```
instead of
```return string("zh-Hans");```
?
Looks reasonable, but with Objective-C who can be sure?